drivers/node/vsphere/vsphere.go

```diff
@@ -268,7 +268,7 @@ func (v *vsphere) getVMFinder() (*find.Finder, error) {
 
 }
 
-//GetCompatibleDatastores get matching prefix datastores
+// GetCompatibleDatastores get matching prefix datastores
```

Comment:
We can discuss this. As of now this will be running in a controlled environment where we know the prefix and have set up datastores in that way accordingly. This could be taken up as future enhancement.

Reply:
[PTX-25457](https://purestorage.atlassian.net/browse/PTX-25457) will be addressing this in future